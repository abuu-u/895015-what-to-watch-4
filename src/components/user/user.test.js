import React from "react";
import renderer from "react-test-renderer";
import User from "./user.jsx";

const authInfo = {
  avatarUrl: `img/1.png`
};

it(`Render User`, () => {
  const tree = renderer
    .create(
        <User
          authInfo={authInfo}
        />
    )
    .toJSON();

  expect(tree).toMatchSnapshot();
});
