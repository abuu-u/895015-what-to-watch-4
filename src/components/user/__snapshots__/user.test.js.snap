// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render User 1`] = `
<div
  className="user-block"
>
  <div
    className="user-block__avatar"
  >
    <a
      href="/mylist"
      onClick={[Function]}
    >
      <img
        alt="User avatar"
        height="63"
        src="https://4.react.pages.academyimg/1.png"
        width="63"
      />
    </a>
  </div>
</div>
`;
